aquadoggo/src/materializer/tasks/dependency.rs

```diff
@@ -75,6 +79,45 @@ pub async fn dependency_task(context: Context, input: TaskInput) -> TaskResult<T
         }
     }
 
+    // Construct additional tasks if the task input matches certain system schemas and all
+    // dependencies have been reduced.
```

Comment:
This bit is in the Whimsical but it was missing from the dependency task (prob because it had nowhere to go without a schema task)

Reply:
Yeh, perfect!